src/ceph-disk/tests/ceph-disk.sh

```diff
@@ -294,6 +294,7 @@ function test_reuse_osd_id() {
     run_mon $dir a || return 1
     run_mgr $dir x || return 1
     create_rbd_pool
+    ceph osd pool create bar 128
```

Comment:
@liewegas why would you want to add more pgs? 

with following patch applied, this test passes:
```diff
diff --git a/src/ceph-disk/tests/ceph-disk.sh b/src/ceph-disk/tests/ceph-disk.sh
index 92ff163f6a..0995e8dd25 100755
--- a/src/ceph-disk/tests/ceph-disk.sh
+++ b/src/ceph-disk/tests/ceph-disk.sh
@@ -294,7 +294,7 @@ function test_reuse_osd_id() {
     run_mon $dir a || return 1
     run_mgr $dir x || return 1
     create_rbd_pool
-    ceph osd pool create bar 128
+    # ceph osd pool create bar 128

     test_activate $dir $dir/dir1 --osd-uuid $(uuidgen) || return 1

@@ -309,7 +309,7 @@ function test_reuse_osd_id() {
     #
     # make sure the OSD is in use by the PGs
     #
-    wait_osd_id_used_by_pgs $osd_id 6 || return 1
+    wait_osd_id_used_by_pgs $osd_id $PG_NUM || return 1
     read_write $dir SOMETHING || return 1

     #
```